### PR TITLE
Update UXP documentation link in the readme

### PR DESCRIPTION
**Pull request by psinghal20**
### Description of your changes
Update documentation link in the readme. The older link was giving 404

PS: Kudos for the release! :smile:

### CODE REVIEW

**Review by hasheddan — COMMENTED** [review]
@psinghal20 thanks for this contribution! Let me know if you need any assistance with the CLA workflow :)

**Comment by psinghal20** [comment]
I actually sent the signed CLA document to `cla@upbound.io`, but I received an error email back. I will try to resend it.

**Comment by hasheddan** [comment]
@psinghal20 thanks for letting me know! I'll make sure this gets fixed ASAP!

**Comment by hasheddan** [comment]
@psinghal20 I believe the issue should now be resolved, could you try and send again? :)

**Comment by psinghal20** [comment]
Sent, please take a look.

**Comment by hasheddan** [comment]
@cla-bot check

File: README.md [hunk 1]
```diff
@@ -121,6 +121,6 @@ If you'd like to upgrade to a specific version, run the following:
 [upbound-cli]: https://github.com/upbound/up
 [create-account]: https://cloud.upbound.io/register
 [login]: https://cloud.upbound.io/login
-[uxp-documentation]: https://cloud.upbound.io/uxp
+[uxp-documentation]: https://cloud.upbound.io/docs/
```
Comment (hasheddan) [inline README.md:124]:
```suggestion
[uxp-documentation]: https://cloud.upbound.io/docs/uxp
```

**Review by hasheddan — COMMENTED** [review]
@psinghal20 CLA bot approved! Mind updating the link to be a little more specific? :)

**Comment by psinghal20** [comment]
Done! @hasheddan thanks for helping out with the CLA.

**Review by hasheddan — APPROVED** [review]

**Pull request merged**